tests/PHPSA/Command/CompileTest.php

```diff
@@ -18,4 +18,27 @@ public function testExecute()
 
         self::assertContains('Memory usage:', $commandTester->getDisplay());
     }
+    // simple as it is
+    public function testIgnorePaths()
```

Comment:
This test will never fail, unless we change the test. We want tests that fail if we change the actual implementation, so you have to use the actual implementation in your test.
A good test could be: run the thing on our fixtures folder (see testExecute() ) with the ignore setting set to ignore one of the 3 folders in fixtures and afterwards check that in the output nothing from the ignored fixtures is seen, but the others are (1 example of each is enough)
But it would probably be better if we had a command option to just overwrite one config setting instead of having to supply a complete config file.

If you want try to implement this, else just remove the test for now.